### PR TITLE
Update keycloak db to turn sslRequired to none

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### Reference issue
https://issues.redhat.com/browse/CRW-765

### What does this PR do:
set sslRequired=none by updating keycloak db

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -78,6 +79,7 @@ require (
 	github.com/onsi/gomega v1.10.0 // indirect
 	github.com/openshift/api v0.0.0-00010101000000-000000000000
 	github.com/operator-framework/operator-sdk v0.0.0-00010101000000-000000000000
+	github.com/pborman/uuid v1.2.0 // indirect
```
Comment (AndrienkoAleksandr) [inline go.mod:82]: Do we need CQ?

Comment (tolusha) [reply go.mod:82]: I am not sure. I didn't add this dependency, go.mod was updated automatically somehow.

Comment (AndrienkoAleksandr) [reply go.mod:82]: VSCode extension could update modify go.mod on the fly...

**Comment by tolusha** [comment]
@davidfestal 
tested for both upstream and downstream
sslRequires is set to none

File: go.mod (hunk 1 above)
Comment (nickboldt) [reply go.mod:82]: if you depend on something new (and therefore so will CRW) then yes, please file a CQ.

Comment (tolusha) [reply go.mod]: I've restored go.mod to prev state.

**Pull request merged**